### PR TITLE
(1487) Access needs further questions design tweaks

### PR DESCRIPTION
**Pull request by pezholio**
This updates some questions and design elements on the access needs further considerations page.

### Before

image

### After

image

### CODE REVIEW

**Comment by pezholio** [comment]
[Access Needs Further Questions design tweaks](https://trello.com/c/NGooPZWV/1487-access-needs-further-questions-design-tweaks)

File: server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ import AccessNeeds from './accessNeeds'
 
 export type AccessNeedsFurtherQuestionsBody = {
   needsWheelchair: YesOrNo
-  mobilityNeeds: string
-  visualImpairment: string
+  healthConditions: YesOrNo
```
Comment (richpjames) [inline server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts:13]: could we use the YesOrNoWithDetail utility type here?

File: server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts [hunk 2]
```diff
@@ -90,8 +90,11 @@ export default class AccessNeedsFurtherQuestions implements TasklistPage {
   response() {
     const response = {
       [this.questions.wheelchair]: sentenceCase(this.body.needsWheelchair),
-      [this.questions.mobilityNeeds]: this.body.mobilityNeeds,
-      [this.questions.visualImpairment]: this.body.visualImpairment,
+      [this.questions.healthConditions]: sentenceCase(this.body.healthConditions),
+    }
+
+    if (this.body.healthConditions === 'yes') {
```
Comment (richpjames) [inline server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts:124]: could we use the `yesOrNoResponseWithDetail` function here?

**Review by richpjames — COMMENTED** [review]
We have some utils that could help with these 'Yes or no with details' responses

File: server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts (hunk 1 above)
Comment (pezholio) [reply server/form-pages/apply/risk-and-need-factors/access-and-healthcare/accessNeedsFurtherQuestions.ts]: Oh course! Nice catch! 👍

**Review by richpjames — APPROVED** [review]

**Pull request merged**